testing/file_test/file_test_base.cpp

```diff
@@ -398,6 +423,7 @@ static auto RunSingleTest(FileTestInfo& test, bool single_threaded,
       output_lock = std::unique_lock<std::mutex>(output_mutex);
     }
 
+    Timer test_timer;
```

Comment:
I wonder whether this timer should be sunk inside the call to `RunSafely` in the multithreaded case, so we're timing the same thing regardless of whether we're running with multiple threads. Then again, it doesn't look to me like `RunSafely` adds much overhead right now so it probably doesn't matter.

Reply:
Done. Note I see no difference in output.